prowler/providers/gcp/gcp_provider.py

```diff
@@ -388,10 +423,11 @@ def test_connection(
         credentials_file: str = None,
         service_account: str = None,
         raise_on_exception: bool = True,
+        provider_id: Optional[str] = None,
```

Comment:
Review if something is calling this function with positional arguments.

Reply:
Done